server/sqlflowserver.go

```diff
@@ -74,6 +86,23 @@ func (s *Server) Run(req *pb.Request, stream pb.SQLFlow_RunServer) error {
 	return nil
 }
 
+func dbConnStrWithCredencial(token, dbConnStr string) string {
+	// TODO(Yancey1989): implement fetch Database crediencial according
+	// to the user token.
+	return dbConnStr
+}
+
+// SessionDB returns a sql.DB object accoding to the request session.
+func sessionDB(session *pb.Session) (*sf.DB, error) {
```

Comment:
sessionDB => `getCachedDBConn`

and should add a global mapping to store the cached connections.

Reply:
done.